### PR TITLE
[ecore_con] Change EAPI symbols for ECORE_CON_API

### PR DESCRIPTION
**Pull request by Coquinho**

### CODE REVIEW

File: src/lib/ecore_con/Ecore_Con.h [hunk 1]
```diff
@@ -96,38 +96,38 @@
 /**
  * @defgroup Ecore_Con_Buffer Ecore Connection Buffering
  * @ingroup Ecore_Con_Group
- *
```
Comment (joaoantoniocardoso) [inline src/lib/ecore_con/Ecore_Con.h:123]: this blank space changes ...

Comment (Coquinho) [reply src/lib/ecore_con/Ecore_Con.h:123]: :(

Comment (Coquinho) [reply src/lib/ecore_con/Ecore_Con.h]: [:(](https://github.com/expertisesolutions/efl/pull/182/commits/68b1fb2fca9d8712c983c619b3b59c22d90dc8ac)

**Comment by Coquinho** [comment]
Hey @walac, there were changes after your approval, so please look again.

**Pull request merged**